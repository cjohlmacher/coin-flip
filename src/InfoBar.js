import React from 'react';

const InfoBar = ({count}) => {
    return (
        <div className="InfoBar">
            <p>Heads: {count.heads}</p>
            <p>Tails: {count.tails}</p>
        </div>
    )
};

InfoBar.defaultProps = {
    count: {heads: 0, tails: 0}
}

export default InfoBar;