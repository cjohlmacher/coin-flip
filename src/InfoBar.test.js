import { render } from '@testing-library/react';
import InfoBar from './InfoBar';

it('renders without crashing', () => {
    render(<InfoBar />);
});

it('matches snapsnot', () => {
    const {asFragment} = render(<InfoBar count={{heads: 3, tails: 3}} />);
    expect(asFragment()).toMatchSnapshot();
});