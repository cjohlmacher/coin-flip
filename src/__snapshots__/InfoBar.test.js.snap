// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapsnot 1`] = `
<DocumentFragment>
  <div
    class="InfoBar"
  >
    <p>
      Heads: 3
    </p>
    <p>
      Tails: 3
    </p>
  </div>
</DocumentFragment>
`;
